src/pages/signin/ResendValidationForm.js

```diff
@@ -80,31 +85,58 @@ class ResendValidationForm extends React.Component {
     }
 
     render() {
+        const isNewAccount = !this.props.account.accountExists && !this.props.account.validated;
```

Comment:
nab - Is it possible for `account.accountExists` to be false and `account.validated` to be true here? Is it necessary to check `.validated` here?

Reply:
No not possible, can simplify it

Reply:
Updated and ready for another look. Thanks!